setup.py

```diff
@@ -30,6 +30,8 @@
         'Programming Language :: Python :: 3.7',
```

Comment:
If we're removing older Python / Django versions from our testing, we should remove the classifiers from this list. We should also add a `python_requires=">=3.8",` line to this config file, so that pip doesn't try to install it against older versions.